spec/rails_helper.rb

```diff
@@ -17,6 +17,7 @@
 ActiveRecord::Migration.maintain_test_schema!
 
 RSpec.configure do |config|
+  config.include Hashable
```

Comment:
This brings Hashable to specs, as it seems a few specs needed it. They were including it at top level, which is really dangerous, as it includes it in **every object** in ruby via the top level object.